newm-server/src/main/resources/static/predefined-genres.json

```diff
@@ -1,4 +1,5 @@
 [
+  "Deprecated. Use /v1/distribution/genres instead.",
```

Comment:
@scandycuz @dmkirshon @escobarjonatan Genres, Roles static hardcoded stuff is deprecated after this merge. This data comes from Eveara now.